src/graphics.rs

```diff
@@ -957,9 +957,14 @@ fn load_shader_internal(
                 &mut max_length as *mut _,
                 error_message.as_mut_ptr() as *mut _,
             );
+            // trim trailing zeros
+            while error_message.last().copied() == Some(0) {
```

Comment:
maybe 
```let error_message = std::string::String::from_utf8_lossy(&error_message[0..max_length as usize]); ``` will do the job of trimming zeroes as well?

Reply:
I think glGetProgramInfoLog does not write max_length in gl.js, but I am not insisting on the implementation - consider this to be an issue ticket :smile:.

Reply:
I also tried using GL_INFO_LOG_LENGTH, but it does not seem to work with a program object

Reply:
oh, ok, my bad :) 
Going to merge it as is and fix gl.js later

Reply:
thank you!

Reply:
Done: https://github.com/not-fl3/miniquad/pull/84